examples/formbot/README.md

```diff
@@ -29,14 +29,21 @@ steps:
     ```
     The model will be stored in the `/models` directory as a zipped file.
 
-2. Test the assistant by running:
+2. Run an instance of [duckling](https://rasa.com/docs/rasa/nlu/components/#ducklinghttpextractor)
+   on port 8000 by either running the docker command
+   ```
+   docker run -p 8000:8000 rasa/duckling
+   ```
+   or installing duckling directly on your machine and starting the server.

```

Comment:
```suggestion
   or [installing duckling](https://github.com/facebook/duckling#quickstart) directly on your machine and starting the server.
```

Reply:
```suggestion
   or [installing duckling](https://github.com/facebook/duckling#requirements) directly on your machine and starting the server.
```
Sure, good idea. changing it to requirements so that they don't miss them